root/static/scripts/edit/URLCleanup.js

```diff
@@ -202,6 +202,11 @@ export const LINK_TYPES = {
     place: '1cd2eb89-2997-4901-87e9-838ac9a68da9',
     series: '4789521b-57b9-4689-9644-46de63190f66',
   },
+  streaming: {
```

Comment:
That should probably be committed apart and refer to [STYLE-1287](https://tickets.metabrainz.org/browse/STYLE-1287).

Reply:
It is commited apart, that's why we have two commits (https://github.com/metabrainz/musicbrainz-server/pull/1189/commits/55320838837b68b65e2d1a3834a1ef33874ffc61)

Reply:
It isn’t, it is in the same commit as changes for Amazon Music. :)

Reply:
Oh, you mean you'd add the rels first and then the Amazon Music bit. But why? They aren't needed up until Amazon Music is added.

Reply:
To improve readability? :) This is the only change related to STYLE-1287.

Reply:
Ok, I think I split it now (I was annoyed because it's hard to split only some of the changes to a file off, but I just discovered git add -p :) )